apiserver/plane/api/urls/issue.py

```diff
@@ -6,9 +6,15 @@
     IssueLinkAPIEndpoint,
     IssueCommentAPIEndpoint,
     IssueActivityAPIEndpoint,
+    WorkspaceIssueAPIEndpoint,
 )
 
 urlpatterns = [
+    path(
+        "workspaces/<str:slug>/issues/<str:project__identifier>-<str:issue__identifier>",
```

Comment:
@MichaelErmer can you please add a trailing `/` here.

Reply:
done